resume-builder/src/App.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './App.css';
+import TestComponent from './TestComponent'
+
+function App() {
```

Comment:
I like how clean the return is done given that you have html tags in the response.

EDIT: I guess similar to my Angular App, on second look it seems like this was auto-generated when you build the application. Shows you how much i know about React :)